dd-sdk-android-ndk/src/main/cpp/utils/signal-monitor.c

```diff
@@ -26,14 +26,14 @@ static sigset_t signals_mask;
 static stack_t signal_stack;
 
 /* the signal handler array */
-static struct sigaction *global_sigaction;
+struct sigaction *volatile datadog_sigaction;
 
 /* the previous signal handler array */
```

Comment:
Is it the previous or the original one ? Either the variable name or the comment is lying.

Reply:
yes need to update the comments...it is our handler (sigaction structure) that substitutes the original one.